src/Microsoft.DotNet.Build.Tasks/Targets/toolruntime.targets

```diff
@@ -17,7 +17,7 @@
     <ToolArchitecture Condition="'$(ToolArchitecture)' == ''">x64</ToolArchitecture>
     <ToolNugetRuntimeId Condition="'$(ToolNugetRuntimeId)' == ''">win7-$(ToolArchitecture)</ToolNugetRuntimeId>
 
-    <ToolRuntimePath Condition="'$(ToolRuntimePath)' == ''">$(BaseOutputPath)$(OSPlatformConfig)\ToolRuntime\</ToolRuntimePath>
+    <ToolRuntimePath Condition="'$(ToolRuntimePath)' == ''">$(IntermediateOutputPath)ToolRuntime\</ToolRuntimePath>
```

Comment:
Ok I'll change it. The reason why I chose that variable is because that si the one defiined in coreclr repo which is the one I used to test my change. I'll change this and then define the RootPath in coreclr repo if it's not defined.